Dockerfile

```diff
@@ -1,14 +1,25 @@
 FROM kbase/sdkbase2:python
 MAINTAINER KBase Developer
 # -----------------------------------------
-RUN apt-get update
-ENV PYTHONUNBUFFERED=True
 
-RUN apt-get install --yes vim
+# the base image really needs an update
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 112695A0E562B32A
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 54404762BBB6E853
+RUN echo "deb http://security.debian.org/debian-security stable-security/updates main" > /etc/apt/sources.list
+# disable the old expired X3 cert and then force an update below
+RUN sed -i '/^mozilla\/DST_Root_CA_X3/s/^/!/' /etc/ca-certificates.conf
```

Comment:
Is this required? Doesn't apt-get upgrade delete DST_Root_CA_X3 automatically?

Reply:
It's been a bit since I did this so I could be wrong, but from what I recall curl would still use that cert unless it was disabled. If you want to be sure just comment out the line and see if the build works I guess

Reply:
I stand corrected. Curl still grabs that root cert without that line.